core/app/models/spree/google_export_option.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  class GoogleExportOption < Base
+    def export
+      Spree::Dependencies.export.constantize.new.export_google_rss(self)
+    end
+
+    def true_keys
```

Comment:
```suggestion
    def enabled_keys
```
I'd rename to `enabled_keys` or `enabled_options` - when I read `true_keys` I don't immediately know what it's referring to.